roles/kubernetes/tasks/cleanup.yml

```diff
@@ -0,0 +1,11 @@
+---
+# This play contains tasks for cleaning up kubernetes
+
+- name: stop kubernetes
+  service: name=kubernetes state=stopped
+
+- name: cleanup iptables for kubernetes
```

Comment:
we should really turn this into a single unit of code in the future; this pattern is all over the place in our code now :(